src/pages/shop/Shop.tsx

```diff
@@ -1,15 +1,193 @@
+                    <ul className='menuList'>
+                      <li>
+                        <Link to='/'>Accessories</Link>
+                      </li>
+                      <li>
+                        <Link to='/'>Backpacks</Link>
+                      </li>
+                      <li className='digital'>
+                        <Link to='/'>Digitals </Link>
+                        <button className='icon' ><MdOutlineKeyboardArrowDown/></button>
+                        <ul  id='subMenu'>
+                          <li>
+                            <Link to='/'>Headphones</Link>
+                          </li>
+                          <li>
+                            <Link to='/'>Speaker</Link>
+                          </li>
+                        </ul>
+                      </li>
+                      <li>
+                        <Link to='/'>Wallets</Link>
+                      </li>
+                      <li>
+                        <Link to='/'>Watches</Link>
+                      </li>
+                      <li>
+                        <Link to='/'>Water Bootles</Link>
+                      </li>
+                      <li>
+                        <Link to='/'></Link>
+                      </li>
+                    </ul>
+                    </div>
+                    
+                  </div>
+                  <div className="productTags">
+                    <p>Product Tags</p>
+                    <Stack direction='column'>
+                           <Wrap spacing={10}>
+                              <WrapItem>
+                                <Button className='btn'><span>Bosch</span></Button>
+                              </WrapItem>
+                              <WrapItem>
+                                <Button className='btn'><span>Corkcicle</span></Button>
+                              </WrapItem>
+                              <WrapItem>
+                                <Button className='btn'><span>Handmade</span></Button>
+                              </WrapItem>
+                              <WrapItem>
+                                <Button className='btn'><span>Kinto</span></Button>
+                              </WrapItem>  <WrapItem>
+                                <Button className='btn'><span>Lether</span></Button>
+                              </WrapItem>  <WrapItem>
+                                <Button className='btn'><span>Lifestyle</span></Button>
+                              </WrapItem>  <WrapItem>
+                                <Button className='btn'><span>Orbitkey</span></Button>
+                              </WrapItem>  <WrapItem>
+                                <Button className='btn'><span>Pet</span></Button>
+                              </WrapItem>  <WrapItem>
+                                <Button className='btn'><span>Soma</span></Button>
+                              </WrapItem>
+                             </Wrap>
```

Comment:
you must have an array of tags (that will be taken from api), so in this case you will have .map function's call for this array